Tests.Integration.Sensitive.Presentation.Web/Token/TokenInvalidationTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.Net;
+using Core.DomainModel.Organization;
+using Newtonsoft.Json.Linq;
+using Tests.Integration.Presentation.Web.Tools;
+using Tests.Integration.Presentation.Web.Tools.Model;
+using Tests.Integration.Sensitive.Presentation.Web.Tools;
+using Tests.Integration.Sensitive.Presentation.Web.Tools.Model;
+using Xunit;
+
+namespace Tests.Integration.Sensitive.Presentation.Web.Token
+{ 
+    public class TokenInvalidationTests : WithAutoFixture
+    {
+        private readonly KitosCredentials _apiUser, _globalAdmin;
+
+        public TokenInvalidationTests()
+        {
+            _apiUser = TestEnvironment.GetCredentials(OrganizationRole.ApiAccess);
+            _globalAdmin = TestEnvironment.GetCredentials(OrganizationRole.GlobalAdmin);
+        }
+
+        [Fact]
+        public void Token_Can_Be_Invalidated_After_Creation()
+        {
```

Comment:
Lav en ekstra bruger i test scripts i stedet for et ekstra projekt + test step. Husk at navngive den korrekt, så man ved, at der ikke skal pilles ved den.-

Reply:
Alternativt skal der oprettes en bruger i testen - specifikt til testen.

Reply:
Måden du gør det på: Debugger i Chrome - opret bruger via kitos og se JSON. Kan genskabes med anonyme objekter + JsonConvert.SerializeObject(..)